pages/registry/cli.mdx

```diff
@@ -1,96 +1,109 @@
 import { Callout } from "nextra-theme-docs";
 
-# `wasmer publish` CLI
-
-`wasmer publish` allows publishing a Wasmer container to the [registry](/registry).
-
-## Usage
-
-### Create a package
-
-Creating a package is very simple, just run:
+# The `package` CLI subcommand
```

Comment:
The package subcommand should go after the publish command.
Publish go first of all